os/binfmt/libelf/libelf_cache.c

```diff
@@ -453,6 +453,12 @@ int elf_cache_init(int filfd, uint16_t offset, off_t filelen, uint8_t compressio
 	number_of_blocks = file_len / cache_blocks_size;
 	elf_compress_type = compression_type;
 
+	/* Set number of blocks to use for caching */
+	if (CONFIG_ELF_CACHE_BLOCKS_COUNT < (CUTOFF_RATIO_CACHE_BLOCKS) * (number_of_blocks))
+		number_blocks_caching = CONFIG_ELF_CACHE_BLOCKS_COUNT;
+	else
+		number_blocks_caching = ((CUTOFF_RATIO_CACHE_BLOCKS) * (number_of_blocks));
```

Comment:
could you use brackets? We guide using them even it has a single statement.

Reply:
Done